$NetBSD: patch-configure,v 1.2 2011/06/03 12:48:48 wiz Exp $

Fix broken test by assuming pkgsrc handles version check for us.
(Problem here is that 1.10 doesn't compare as bigger than 1.4.)
http://tickets.musicbrainz.org/browse/OTHER-44

--- configure.orig	2006-11-28 20:34:18.000000000 +0000
+++ configure
@@ -21634,7 +21634,7 @@ fi
     { echo "$as_me:$LINENO: checking for taglib >= 1.4" >&5
 echo $ECHO_N "checking for taglib >= 1.4... $ECHO_C" >&6; }
         VERSION_CHECK=`expr $TAGLIB_VERSION \>\= 1.4`
-        if test "$VERSION_CHECK" = "1" ; then
+        if test "1" = "1" ; then
             { echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6; }
             succeeded=yes
